### DISCUSSION TITLE
How to prevent window from closing on blur?

### DISCUSSION THREAD

**Discussion by brettshepherd** [johnlindquist/kit#701] [Q&A] [answered]
Sorry if this has been asked before, but I couldn't find it. Basically I have a series of `arg()` calls with user input that shows the user more and more detailed options.

To get the `arg()` input text,  a user has to interact with another program, but every time you click off the Script Kit window it closes down and you have to start over. 

I tried `setIgnoreBlur(true)`, but that had no effect. 

I notice that the `show()` function window stays open regardless of blur but that takes html and isn't suitable for this. Is there no way to have the same effect of not closing on the default window?

**Comment by johnlindquist** [comment, marked as answer]
`arg` can take a config object as the first argument:


[Open ignore-blur-demo in Script Kit](https://scriptkit.com/api/new?name=ignore-blur-demo&url=https://gist.githubusercontent.com/johnlindquist/53ac0a1b7ff2361f7b522f3b72a19011/raw/85b01f219c4947f6ed11ee1ba7f4ca3c66a7c977/ignore-blur-demo.ts")

```js
// Name: Ignore Blur Demo

import "@johnlindquist/kit"

await arg({
    placeholder: "This should stay open",
    ignoreBlur: true
}, ["Yay", "Huzzah!"])


```
